libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -2542,13 +2540,8 @@ void IRGeneratorForStatements::appendBareCall(
 	}
 
 	templ("success", IRVariable(_functionCall).tupleComponent(0).name());
-	if (IRVariable(_functionCall).tupleComponent(1).type().category() == Type::Category::InaccessibleDynamic)
-		templ("returndataVar", "");
-	else
-	{
-		templ("returndataVar", IRVariable(_functionCall).tupleComponent(1).part("mpos").name());
-		templ("extractReturndataFunction", m_utils.extractReturndataFunction());
-	}
+	templ("returndataVar", IRVariable(_functionCall).tupleComponent(1).commaSeparatedList());
+	templ("extractReturndataFunction", m_utils.extractReturndataFunction());
```

Comment:
The `extractReturndataFunction` already has a special case if there is no `returndatacopy`: It returns the empty byte array.

Reply:
Note to self: For EVM versions that do not support return data e.g., spurious dragon, empty byte array is returned.

Reply:
@chriseth So for evm versions that did not have return data, how did the caller figure out if a bare call is a success or not?